### DISCUSSION TITLE
Hold action button script

### DISCUSSION THREAD

**Discussion by ArecKyura** [kalkih/mini-media-player#724] [Q&A]
I am trying to launch a script when I hold down a button, but if I enter "hold_action", it doesn't work, or rather, the code doesn't give errors but if I hold down the key and then release it, it is detected as a single touch. It seems that the Mini media player does not have the built-in ability to hold down keys. This is my code (working):

type: custom:mini-media-player
name: Musica
source: icon
sound_mode: full
entity: media_player.echo_dot_camera_da_letto
info: full
shortcuts:
  columns: 3
  buttons:
    - name: Giappominchiate
      type: service
      id: spotcast.start
      data:
        device_name: Intera casa
        uri: spotify:playlist:1k0UT1fO9Owvwbq4w6L0f8
    - name: Tagliavene
      type: service
      id: spotcast.start
      data:
        device_name: Intera casa
        uri: spotify:playlist:37i9dQZF1EVKuMoAJjoTIw
  ---    ### **hold_action: example** ----
    - name: Good Vibe
      type: service
      id: spotcast.start
      data:
        device_name: Intera casa
        uri: spotify:playlist:37i9dQZF1DX9qUVnHy7VXV
        random_song: true
        shuffle: true
volume_stateless: false
group: false
icon_image: /local/immagini/Echodotpotente.png

**Comment by rgerbranda** [comment]
Same problem here

**Reply by adri123** [reply]
I cannot make it work too
